src/test/kotlin/no/nav/medlemskap/regler/v1/RegelsettForGrunnforordningenTest.kt

```diff
@@ -16,6 +16,16 @@ class RegelsettForGrunnforordningenTest {
         assertEquals(Resultattype.JA, evaluer(personleser.enkelNorsk()))
     }
 
+    @Test
+    fun `person med flere statsborgerskap i periode får nei`() {
+        assertEquals(Resultattype.NEI, evaluer(personleser.norskMedFlereStatsborgerskap()))
+    }
+
+    @Test
+    fun `person med flere statsborgerskap som overlapper periode får nei`() {
```

Comment:
Flott at du legger til tester! Vurder gjerne om du kan legge til enda flere, for å sjekke ulike edge cases. Dekker vi "periode inni annen periode", "periode rundt annen periode", "periode som overlapper annen periode", osv.. ?